android/kroll-apt/src/main/resources/org/appcelerator/kroll/annotations/generator/ProxyBinding.fm

```diff
@@ -755,8 +755,12 @@ beprovided to handle the result.
 		holder = holder->FindInstanceInPrototypeChain(getProxyTemplate(isolate));
 	}
 	if (holder.IsEmpty() || holder->IsNull()) {
-		LOGE(TAG, "Couldn't obtain argument holder");
-		args.GetReturnValue().Set(v8::Undefined(isolate));
-		return;
+		if (!moduleInstance.IsEmpty()) {
+			holder = moduleInstance.Get(isolate);
```

Comment:
seems like you'd need to do another `if` check like above here after trying the module instance? I think it *shouldn't* ever be empty/null, but who knows. We don't want to simply hard crash if it ever is...